projects/1.16/assets/forbidden-arcanus/forbidden_arcanus/lang/zh_cn.json

```
@@ -0,0 +1,361 @@
+    "item.forbidden_arcanus.obsidian_hoe": "黑曜石锄",
+    "item.forbidden_arcanus.battle_skull": "战骨",
+    "item.forbidden_arcanus.bone_sword": "骨剑",
+    "item.forbidden_arcanus.bone_shovel": "骨锹",
+    "item.forbidden_arcanus.bone_pickaxe": "骨镐",
+    "item.forbidden_arcanus.bone_axe": "骨斧",
+    "item.forbidden_arcanus.bone_hoe": "骨锄",
+    "item.forbidden_arcanus.mystical_dagger": "神秘匕首",
+    "item.forbidden_arcanus.draco_arcanus_helmet": "天龙神秘头盔",
+    "item.forbidden_arcanus.draco_arcanus_chestplate": "天龙神秘胸甲",
+    "item.forbidden_arcanus.draco_arcanus_leggings": "天龙神秘护腿",
+    "item.forbidden_arcanus.draco_arcanus_boots": "天龙神秘靴子",
+    "item.forbidden_arcanus.tyr_helmet": "战神提尔之头盔",
+    "item.forbidden_arcanus.tyr_chestplate": "战神提尔之胸甲",
+    "item.forbidden_arcanus.tyr_leggings": "战神提尔之护腿",
+    "item.forbidden_arcanus.tyr_boots": "战神提尔之靴子",
+    "item.forbidden_arcanus.mortem_helmet": "亡骨头盔",
+    "item.forbidden_arcanus.mortem_chestplate": "亡骨胸甲",
+    "item.forbidden_arcanus.mortem_leggings": "亡骨护腿",
+    "item.forbidden_arcanus.mortem_boots": "亡骨靴子",
+    "item.forbidden_arcanus.arcane_gold_helmet": "神秘金头盔",
+    "item.forbidden_arcanus.arcane_gold_chestplate": "神秘金胸甲",
+    "item.forbidden_arcanus.arcane_gold_leggings": "神秘金护腿",
+    "item.forbidden_arcanus.arcane_gold_boots": "神秘金靴子",
+    "item.forbidden_arcanus.obsidian_helmet": "黑曜石头盔",
+    "item.forbidden_arcanus.obsidian_shoulder_pads": "黑曜石护肩",
+    "item.forbidden_arcanus.obsidian_knee_pads": "黑曜石护膝",
+    "item.forbidden_arcanus.obsidian_boots": "黑曜石靴子",
+    "item.forbidden_arcanus.boom_arrow": "爆炸箭矢",
+    "item.forbidden_arcanus.draco_arcanus_arrow": "神秘天龙箭矢",
+
+    "item.forbidden_arcanus.sanity_meter": "理智表",
+    "item.forbidden_arcanus.lens_of_veritatis": "真理透镜",
+    "item.forbidden_arcanus.purifying_soap": "纯化肥皂",
+    "item.forbidden_arcanus.wet_purifying_soap": "湿润的纯化肥皂",
+    "item.forbidden_arcanus.obsidian_skull_shield": "黑曜石头骨盾牌",
+    "item.forbidden_arcanus.aureal_bottle": "耀金瓶",
+    "item.forbidden_arcanus.arcane_crystal_dust_speck": "神秘水晶粉末颗粒",
+    "item.forbidden_arcanus.arcane_bone_meal": "神秘骨粉",
+    "item.forbidden_arcanus.zombie_arm": "僵尸手臂",
+    "item.forbidden_arcanus.shiny_zombie_arm": "闪光僵尸手臂",
+    "item.forbidden_arcanus.test_tube": "试管",
+    "item.forbidden_arcanus.blacksmith_gavel_head": "工匠锤头",
+    "item.forbidden_arcanus.wooden_blacksmith_gavel": "木工匠锤",
+    "item.forbidden_arcanus.stone_blacksmith_gavel": "石工匠锤",
+    "item.forbidden_arcanus.golden_blacksmith_gavel": "金工匠锤",
+    "item.forbidden_arcanus.iron_blacksmith_gavel": "铁工匠锤",
+    "item.forbidden_arcanus.diamond_blacksmith_gavel": "钻石工匠锤",
+    "item.forbidden_arcanus.netherite_blacksmith_gavel": "下界合金工匠锤",
+    "item.forbidden_arcanus.arcane_golden_blacksmith_gavel": "神秘金工匠锤",
+    "item.forbidden_arcanus.reinforced_arcane_golden_blacksmith_gavel": "强化神秘金工匠锤",
+    "item.forbidden_arcanus.obsidian_blacksmith_gavel": "黑曜石工匠锤",
```

```suggestion
    "item.forbidden_arcanus.blacksmith_gavel_head": "铁匠锤头",
    "item.forbidden_arcanus.wooden_blacksmith_gavel": "木铁匠锤",
    "item.forbidden_arcanus.stone_blacksmith_gavel": "石铁匠锤",
    "item.forbidden_arcanus.golden_blacksmith_gavel": "金铁匠锤",
    "item.forbidden_arcanus.iron_blacksmith_gavel": "铁铁匠锤",
    "item.forbidden_arcanus.diamond_blacksmith_gavel": "钻石铁匠锤",
    "item.forbidden_arcanus.netherite_blacksmith_gavel": "下界合金铁匠锤",
    "item.forbidden_arcanus.arcane_golden_blacksmith_gavel": "神秘金铁匠锤",
    "item.forbidden_arcanus.reinforced_arcane_golden_blacksmith_gavel": "强化神秘金铁匠锤",
    "item.forbidden_arcanus.obsidian_blacksmith_gavel": "黑曜石铁匠锤",
```
Blacksmith应为`铁匠`。
至于`铁匠锤`和`铁铁匠锤`会不会很怪……这个译名早已有之，如BetterEnd。

---

要不叫`锻工锤`？和`锻炉`配合一下